tests/knative-serving/knative_serving_test.go

```diff
@@ -77,7 +78,12 @@ func TestKnativeServingAcceptance(t *testing.T) {
 		},
 	}
 
-	if _, err = serviceClient.Create(&svc); err != nil {
+	_, err = serviceClient.Create(&svc)
+
+	switch {
+	case errors.IsAlreadyExists(err):
```

Comment:
But why does the knative service still exists? Isn't the test supposed to do a cleanup of the knative service if the test finishes? 🤔

Reply:
Yes it should, but for some reason it failed to do the cleanup. I thought it might be an RBAC issue, but it is not.